birdhouse/components/weaver/default.env

```diff
@@ -63,7 +65,8 @@ export WEAVER_IMAGE_URI='registry.hub.docker.com/${WEAVER_IMAGE}'
 # default release of the MongoDB version employed by Weaver
 # NOTE:
 #   MongoDB>=5.0 is REQUIRED for Weaver>=4.5.0
-export WEAVER_MONGODB_VERSION=5.0.4
+#   MongoDB==7.x works, but default remains 5.0 to avoid DB migration issues (update manually as desired)
```

Comment:
There is no migration needed for Weaver's data by itself. It still uses the same structure.
This is just a note in case a MongoDB migrations itself *might* be needed.  I have not tried an actual migration myself. I just booted a new v7 on my server and it works. I simply don't want this component to become responsible for migrating (if any needed) or any issue it could incur if I simply bumped the number, since it is irrelevant for Weaver itself.